neo/Consensus/ConsensusService.cs

```diff
@@ -398,7 +398,7 @@ private void OnPrepareRequestReceived(ConsensusPayload payload, PrepareRequest m
             if (context.RequestSentOrReceived() || context.NotAcceptingPayloadsDueToViewChanging()) return;
             if (payload.ValidatorIndex != context.PrimaryIndex || message.ViewNumber != context.ViewNumber) return;
             Log($"{nameof(OnPrepareRequestReceived)}: height={payload.BlockIndex} view={message.ViewNumber} index={payload.ValidatorIndex} tx={message.TransactionHashes.Length}");
-            if (message.Timestamp <= context.PrevHeader().Timestamp || message.Timestamp > TimeProvider.Current.UtcNow.AddMinutes(10).ToTimestamp())
+            if (message.Timestamp <= context.PrevHeader().Timestamp || message.Timestamp > TimeProvider.Current.UtcNow.AddMinutes(2).ToTimestamp())
```

Comment:
Quick question: what would happen if the network is actually congested and we need more than 2 minutes to generate a  block?

Reply:
View change, like now, but not in generate the block, in receive the PrepareRequest, it's different